components/Avatar/Avatar.js

```diff
@@ -32,7 +33,7 @@ export function Avatar({
   }
 
   return (
-    <Link href={`/stories/${user.user_id}`}>
+    <Link href={link}>
```

Comment:
We discussed this one.

Reply:
This code is from an old commit.
This issue was fixed in a newer commit.
https://github.com/alshakerM/my-instagram/pull/83/commits/767952d3d093835b725e014942742c609831abc0